### PR TITLE
remove some event_loop() fixtures

### PR DESCRIPTION
**Pull request by altendky**
Draft for:
- [x] Seeing if tests still work...
- [x] Does this slow tests down significantly?
- [ ] ~Removal of `sys.exit()` from the daemon `.exit()` method offered separately as https://github.com/Chia-Network/chia-blockchain/pull/10454.~
- [ ] ~Some test fixes being proposed in https://github.com/Chia-Network/chia-blockchain/pull/10442 are included here.~
- [x] Increased runtimes for the full node tests are likely unacceptable.  The previous workflow timeout was 40 minutes.  Observed maximum runtimes from a sample of one build trigger are.
  - macOS: still cancelled at 70 minute timeout
  - Ubuntu: 66 minutes
- [x] Includes changes from https://github.com/Chia-Network/chia-blockchain/pull/10504, merge it first.

### CODE REVIEW

**Comment by arvidn** [comment]
I think it would make sense to break out the `async` -> `sync` fixture changes into a separate PR.

**Comment by altendky** [comment]
Separated in https://github.com/Chia-Network/chia-blockchain/pull/10504.

File: tests/core/full_node/test_mempool.py [hunk 1]
```diff
@@ -80,19 +79,6 @@ def generate_test_spend_bundle(
...
-@pytest.fixture(scope="module")
-def event_loop():
-    loop = asyncio.get_event_loop()
-    yield loop
-
-
-# TODO: this fixture should really be at function scope, to make all tests independent.
-# The reason it isn't is that our simulators can't be destroyed correctly, which
-# means you can't instantiate more than one per process, so this is a hack until
-# that is fixed. For now, our tests are not independent
```
Comment (arvidn) [inline tests/core/full_node/test_mempool.py:95]: I believe this comment is still relevant

Comment (altendky) [reply tests/core/full_node/test_mempool.py:95]: https://gist.github.com/altendky/8128bf09b153034a38edf09f60a111c1 It looks like this now runs ok with the fixture set to function scope.  I left it at module scope for later consideration of performance tradeoffs.

**Review by arvidn — APPROVED** [review]

**Pull request merged**